autorest/codegen/templates/templates_package_dataplane/README.md.jinja2

```diff
@@ -0,0 +1,91 @@
+{% set package_pprint_name = code_model.options['package_pprint_name'] -%}
+{% set package_name = code_model.options['package_name'] -%}
+{% set namespace = code_model.options['namespace'] -%}
+{% set client_name = code_model.options['client_name'] -%}
+# {{ package_pprint_name }} client library for Python
+<!-- write necessary description of service -->
+
+## _Disclaimer_
+
+_Azure SDK Python packages support for Python 2.7 has ended 01 January 2022. For more information and questions, 
+please refer to https://github.com/Azure/azure-sdk-for-python/issues/20691_
+
+## Getting started
+
+### Installating the package
+
+```bash
+python -m pip install {{ package_name }}
+```
+
+#### Prequisites
+
+- Python 3.7+ is required to use this package.
+- You need an [Azure subscription][azure_sub] to use this package.
+- An existing {{ package_pprint_name }} instance.
+
+#### Create with an Azure Active Directory Credential
```

Comment:
you can't be sure the package uses AAD, you can look at the credential scope policy property / other credential properties on the code model to see what kind of credentials we use